src/components/form/skill/AddSkill.js

```diff
@@ -127,9 +127,15 @@ const AddSkill = ({ onClose, isEdit, values }) => {
 
   const getInitialValues = () => {
     if (isEdit) {
+      const skills = data.get.skills;
+
+      const index = skills.findIndex(skill => {
```

Comment:
```suggestion
const index = skills.findIndex(skill => skill.id === values.id)
```
We can remove the `return`

Reply:
done [database-integrate 4ff98e8] database-integrate:#7 - remove return from findIndex